src/Auth0.php

```diff
@@ -611,11 +611,11 @@ public function setAccessTokenExpiration(
      * Get the specified parameter from POST or GET, depending on configured response mode.
      *
      * @param string $parameterName Name of the parameter to pull from the request.
-     * @param int $filter Defaults to FILTER_SANITIZE_STRING. The type of PHP filter_var() filter to apply.
+     * @param int $filter Defaults to \FILTER_UNSAFE_RAW. The type of PHP filter_var() filter to apply.
```

Comment:
PHP's FILTER_SANITIZE_STRING was deprecated in 8.1 in favor of FILTER_UNSAFE_RAW. FILTER_UNSAFE_RAW is PHP's native, default sanitization filter and works largely the same as FILTER_SANITIZE_STRING. [Related Internals discussion.](https://externals.io/message/113657#114079)